compiler/src/iree/compiler/Codegen/Common/MaterializeEncodingIntoNop.cpp

```diff
@@ -41,7 +41,8 @@ struct MaterializeEncodingIntoNopPass
     };
 
     RewritePatternSet materializeEncodingPattern(context);
-    MaterializeEncodingTypeConverter typeConverter(materializeEncodingFn);
+    MaterializeEncodingTypeConverter typeConverter(materializeEncodingFn,
+                                                   false);
```

Comment:
nit: add function argument comment to `false`. Same question: do we really need to pass the information to type converter?

```suggestion
                                                   /*...=*/false);
```